custom-tests.json

```
@@ -838,7 +838,7 @@
       "__base": "var instance = navigator.plugins;"
     },
     "ProcessingInstruction": {
-      "__base": "var instance = document.createProcessingInstruction('xml', \"version='1.0' encoding='UTF-8'\");"
+      "__base": "var instance = document.createProcessingInstruction('bcd', \"version='1.0' encoding='UTF-8'\");"
```

```suggestion
      "__base": "var instance = document.createProcessingInstruction('xml-stylesheet', 'type=\"text/xsl\" href=\"transform.xsl\"');"
```

This produces the second example from https://developer.mozilla.org/en-US/docs/Web/XML/XML_introduction#displaying_xml